compiler/main/docsDriver.cpp

```diff
@@ -17,10 +17,12 @@
  * limitations under the License.
  */
 
+#include "commonFlags.h"
+#include "driver.h"
 #include "docsDriver.h"
```

Comment:
I believe it's a good practice to ensure that the #include for the current implementation file appears first i.e.
#include "docsDriver.h" should appear first in this case.  This ensures that the header file does not accidentally
rely on any .h files that happen to be included first elsewhere.  This in turn makes it less likely that a change in
one header file will ripple unpredictably across the system.